Dockerfile

```
@@ -0,0 +1,8 @@
+FROM node:latest
+
+WORKDIR /usr/app
+
+COPY . .
+RUN npm i
```

```suggestion
COPY package.json .
RUN npm i
COPY . .
```
installing the dependencies in extra layer allows docker to cache them when they do not change